test/include/graph_test/graph_test.h

```diff
@@ -162,5 +171,18 @@ class DBTest : public BaseGraphTest {
     }
 };
 
+// This class starts multiple database in on-disk mode.
+class MultipleDBTest : public BaseGraphTest {
+public:
+    inline void runTest(std::vector<std::unique_ptr<TestStatement>>& statements, bool flag) {
+        for (auto& statement : statements) {
+            auto conn_name = *statement->conn_name;
+            createConn(conn_name);
+            if (flag)
```

Comment:
Is this flag duplicate with the initFlag in `initGraphIfNecessary`?

Reply:
this flag respects the dataset is not null, while the initFlag in `initGraphIfNecessary` respects whether the database inits the graph